code/game/machinery/quantum_pad.dm

```diff
@@ -178,6 +178,8 @@
 						continue
 
 				do_teleport(ROI, get_turf(target_pad),null,TRUE,null,null,null,null,TRUE)
+				if(prob(5))
+					new /obj/item/reagent_containers/food/snacks/store/bread/plain/teleporter(get_turf(target_pad)) //I have done nothing but teleport bread for the past 3 days
```

Comment:
5% chance per thing being teleported to just create bread from nothing?

Reply:
As part of the TF2 short that this PR is based on, teleporters would spawn bread ingame: https://www.youtube.com/watch?v=td_0i1hR8s8

The chance can be lowered though, since this is just supposed to be a rare easter egg and not a "proper" gameplay mechanic.

Reply:
We don't need to have it just like tf2

Reply:
I mean, quantum teleporting could just turn any bread you teleport into the green, mutated kind. Instead of magically creating bread out of nowhere.

Reply:
https://www.youtube.com/watch?v=kwYjV_zry6Q